ocaml/xapi/test_vdi_allowed_operations.ml

```diff
@@ -379,6 +419,50 @@ let test_operations_restricted_during_rpu =
   ; "test_update_allowed_operations" >:: test_update_allowed_operations
   ]
 
+(* Xapi_vdi.check_operation_error should not throw an
+   exception in case of invalid references, as it is called from
+   the message forwarding layer. *)
+let test_null_vm =
+  let all_vdi_operations =
```

Comment:
Is it worth moving this in the `vdi` module so that we have an official place with all the vdi operations if we need to refer to it?

Reply:
Good idea, I moved it into `Xapi_vdi_helpers` as `all_ops`, following the pattern of `Xapi_vbd_helpers` and `Xapi_vif_helpers`, which both have an `all_ops` variable.